src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -159,6 +158,19 @@
                       (error-handler error)
                       (log/error :write-file-failed error)))))))))
 
+(defn- get-file-path [dir path]
+  (let [[dir path'] (map #(-> (when %
+                                (string/replace % "///" "/"))
+                              js/decodeURI)
+                         [dir path])
+        path (if (string/starts-with? path' dir)
```

Comment:
If path' or dir are nil what should the behavior be? string fns shouldn't take nils, which the previous binding allows

Reply:
As stated above, `dir` will never be a nil.